crates/iota-grpc-api/tests/checkpoint_stream.rs

```diff
@@ -545,7 +545,9 @@ async fn test_future_end_sequence_number_only_full() {
         match res {
             Ok(cp) => match iota_grpc_api::client::GrpcNodeClient::deserialize_checkpoint(&cp) {
                 Ok(iota_grpc_api::client::CheckpointContent::Data(checkpoint_data)) => {
-                    result.push(checkpoint_data.checkpoint_summary.sequence_number);
+                    if let Some(v1_data) = checkpoint_data.as_v1() {
```

Comment:
Should we rather use a `match` here, so we don't miss these places when we add new types?

Reply:
Thanks for pointing this out! Fixed it at [0469376](https://github.com/iotaledger/iota/pull/7511/commits/0469376723d56074ca9fa703d333e4a9cd104fc8).